src/main/java/io/vertx/core/http/HttpServerOptions.java

```diff
@@ -169,6 +174,7 @@
   private boolean webSocketPreferredClientNoContext;
   private int webSocketClosingTimeout;
   private TracingPolicy tracingPolicy;
+  private String pcapCaptureFile;
```

Comment:
this should be removed in favor of `NetworkOptions `

Reply:
Fixed